renovate.json

```diff
@@ -14,6 +14,9 @@
 		"dependencies",
 		"3. to review"
 	],
+	"commitMessageAction": "Bump",
+	"commitMessageTopic": "{{depName}}",
+	"commitMessageExtra": "from {{currentVersion}} to {{#if isPinDigest}}{{{newDigestShort}}}{{else}}{{#if isMajor}}{{prettyNewMajor}}{{else}}{{#if isSingleVersion}}{{prettyNewVersion}}{{else}}{{#if newValue}}{{{newValue}}}{{else}}{{{newDigestShort}}}{{/if}}{{/if}}{{/if}}{{/if}}",
```

Comment:
I could imagine `currentVersion` to be undefined in some cases (pinned digests). We'll see how it behaves in practice. The official documentation does not elaborate further on the value of this variable.

Reply:
I took inspiration from https://github.com/DefectDojo/django-DefectDojo/blob/master/.github/renovate.json

Reply:
![Bildschirmfoto vom 2023-02-24 10-38-53](https://user-images.githubusercontent.com/1374172/221144994-a16219f9-99a3-47d9-b65c-e4371e2e62c4.png)

seems to work ok